test/capi/capiLinearGradient.cpp

```diff
@@ -135,3 +139,37 @@ TEST_CASE("Linear Gradient spread", "[capiLinearGradient]")
     REQUIRE(tvg_gradient_del(gradient) == TVG_RESULT_SUCCESS);
     REQUIRE(tvg_gradient_del(NULL) == TVG_RESULT_INVALID_ARGUMENT);
 }
+
+TEST_CASE("Stroke Linear Gradient", "[capiLinearGradient]")
+{
+    Tvg_Paint *shape = tvg_shape_new();
+    REQUIRE(shape != NULL);
```

Comment:
Please keep the code consistency.
REQUIRE(shape);

Reply:
done